package.json

```diff
@@ -30,9 +30,11 @@
   "dependencies": {
     "async": "^0.9.x",
     "jsonld": "^0.4.5",
+    "mz": "^2.6.0",
```

Comment:
Yeah, I do hate that `fs-extra` doesn't offer a native promise-based interface (and instead outsources that to promisify). :(
Ok, so, might as well keep `mz` (though move it to devDependencies), and remove the Bluebird dep.